src/Common/NamePrompter.h

```diff
@@ -115,4 +125,6 @@ class IHints
     NamePrompter<MaxNumHints> prompter;
 };
 
+void appendHintsString(String & message, const std::vector<String> & hints, const String & name);
+
```

Comment:
do we need this? I don't see any implementation or usage...

Reply:
whoops, good catch, not sure how it went through.